rskj-core/src/test/java/co/rsk/peg/bitcoin/P2shP2wshErpCustomRedeemScriptBuilderTest.java

```diff
@@ -0,0 +1,196 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.script.Script;
+import co.rsk.bitcoinj.script.ScriptOpCodes;
+import co.rsk.peg.constants.BridgeConstants;
+import co.rsk.peg.constants.BridgeMainNetConstants;
+import com.google.common.collect.Lists;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class P2shP2wshErpCustomRedeemScriptBuilderTest {
+
+    private static final BridgeConstants bridgeMainnetConstants = BridgeMainNetConstants.getInstance();
+    private static final long CSV_VALUE = bridgeMainnetConstants.getFederationConstants().getErpFedActivationDelay();
+    private static final List<BtcECKey> emergencyKeys = BitcoinTestUtils.getBtcEcKeysFromSeeds(
+        new String[]{"fb01", "fb02", "fb03", "fb04"}, true
+    );
+    private static final int ERP_THRESHOLD = emergencyKeys.size() / 2 + 1;
+    private static final P2shP2wshCustomErpRedeemScriptBuilder builder = P2shP2wshCustomErpRedeemScriptBuilder.builder();
+    private static final List<BtcECKey> defaultKeys = BitcoinTestUtils.getBtcEcKeysFromSeeds(
+        new String[]{"fa00","fa01", "fa02", "fa03", "fa04", "fa05", "fa06", "fa07", "fa08", "fa09",
+            "fa10","fa11", "fa12", "fa13", "fa14", "fa15", "fa16", "fa17", "fa18", "fa19",
+        }, true
+    );
+    private static final int DEFAULT_THRESHOLD = defaultKeys.size() / 2 + 1;
+
+    @Test
+    void of_withTwoScripts_withTheSameDefaultPubKeys_withDifferentOrder_shouldSortTheKeysAndTurnThemEquals() {
```

Comment:
```suggestion
    void of_withTwoScripts_withTheSameKeys_withDifferentOrder_shouldSortTheKeysAndTurnThemEquals() {
```

not using the default ones right? jaja